bin/express-cli.js

```diff
@@ -467,7 +467,7 @@ function main (options, done) {
     if (options.view === true) {
       warning('the default view engine will not be jade in future releases\n' +
```

Comment:
Unnecessary warning if `jage` is no longer the default.

Reply:
Should I remove this warning completely?